sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2885,6 +2905,12 @@ class SQLConf extends Serializable with Logging {
     LegacyBehaviorPolicy.withName(getConf(SQLConf.LEGACY_TIME_PARSER_POLICY))
   }
 
+  def coalesceBucketsInSortMergeJoinEnabled: Boolean =
+    getConf(COALESCE_BUCKETS_IN_SORT_MERGE_JOIN_ENABLED)
+
+  def coalesceBucketsInSortMergeJoinMaxNumBucketsDiff: Int =
```

Comment:
nit: we usually don't add methods if the config is only used once. just inline it.